### PR TITLE
Fix clang-13 warnings

### PR DESCRIPTION
**Pull request by aobolensk**

### CODE REVIEW

**Review by dmitryermilov — APPROVED** [review]

File: _studio/mfx_lib/vpp/src/mfx_vpp_sw_internal.cpp [hunk 1]
```diff
@@ -40,10 +40,6 @@
 #include "mfx_procamp_vpp.h"
 #include "mfx_detail_enhancement_vpp.h"
 
-#if defined(MFX_ENABLE_VPP) && defined(MFX_ENABLE_IMAGE_STABILIZATION_VPP)
-#include "mfx_image_stabilization_vpp.h"
```
Comment (dvrogozh) [inline _studio/mfx_lib/vpp/src/mfx_vpp_sw_internal.cpp:44]: Why do we leave stale files in the repo? This header and https://github.com/intel-innersource/drivers.gpu.media.sdk.lib/blob/master/_studio/mfx_lib/vpp/src/mfx_image_stabilization_vpp.cpp?

IMHO, this should be SEPARATE commit to delete image stabilization!

File: builder/FindGlobals.cmake [hunk 2]
```diff
@@ -265,6 +265,9 @@ if(NOT CMAKE_SYSTEM_NAME MATCHES Windows)
         -Wno-unknown-pragmas
         -Wno-unused
       >
+      $<$<AND:$<CXX_COMPILER_ID:Clang>,$<VERSION_GREATER_EQUAL:${CMAKE_CXX_COMPILER_VERSION},13.0>>:
```
Comment (dvrogozh) [inline builder/FindGlobals.cmake:268]: Can you, please, try to remove `no-unused` instead? If there are just few new hits - easier to fix them. If that's something more global - then let's do your trick here.

**Review by dvrogozh — CHANGES_REQUESTED** [review]

File: builder/FindGlobals.cmake (hunk 2 above)
Comment (dvrogozh) [reply builder/FindGlobals.cmake]: drop from this commit

Comment (aobolensk) [reply builder/FindGlobals.cmake]: Dropped this line from first commit.
Added commit that fixes -Wunused warnings

File: _studio/mfx_lib/vpp/src/mfx_vpp_utils.cpp [hunk 3]
```diff
@@ -1711,12 +1700,6 @@ size_t GetConfigSize( mfxU32 filterId )
         {
             return sizeof(mfxExtVPPFrameRateConversion);
         }
-#if defined(MFX_ENABLE_IMAGE_STABILIZATION_VPP)
```
Comment (dvrogozh) [inline _studio/mfx_lib/vpp/src/mfx_vpp_utils.cpp:1714]: belongs to next commit

Comment (aobolensk) [reply _studio/mfx_lib/vpp/src/mfx_vpp_utils.cpp:1714]: Moved to next commit

**Review by dvrogozh — APPROVED** [review]

**Review by dmitryermilov — APPROVED** [review]
